app/controllers/support_interface/application_forms/courses_controller.rb

```diff
@@ -45,26 +45,6 @@ def create
         end
       end
 
-      def edit
-        @change_course_choice = ChangeCourseChoiceForm.new
-      end
-
-      def update
-        @change_course_choice = ChangeCourseChoiceForm.new(change_course_choice_params)
-
-        begin
-          if @change_course_choice.save(params[:application_choice_id])
-            flash[:success] = 'Course successfully changed'
-            redirect_to support_interface_application_form_path(application_form_id)
-          else
-            render :edit
-          end
-        rescue CourseChoiceError, ProviderInterviewError => e
```

Comment:
could we remove the course choice and the provider interview error as I don't think we use them anywhere?